app/views/dor_harvester/_form.html.erb

```diff
@@ -5,10 +5,13 @@
     <div class="card">
       <div class="card-header" role="tab" id="sdr-item-submission">
         <h4 class="mb-0">
-          <button data-toggle="collapse" href="#sdr-item-submission-section" aria-expanded="true" aria-controls="sdr-item-submission-section" class="btn btn-link">
-            <%= t('admin.items.item-submission') %>
-            <span class="glyphicon glyphicon-chevron-right float-right"></span>
-            <span class="glyphicon glyphicon-chevron-down float-right"></span>
+          <button data-toggle="collapse" href="#sdr-item-submission-section" aria-expanded="true" aria-controls="sdr-item-submission-section" class="btn btn-link w-100">
```

Comment:
Because we want the button to be the whole size, we have to do a float-left, float-right thing. I guess we could flex inside the button as an alternative? But that also felt weird.

Reply:
Does `.stretched-link` help?

Reply:
It seems like we can do that, but then have some kind of sibling css selector. 🤷‍♂ do you have a preference?

Reply:
I've made the `.stretched-link` change a little cleaner.. hopefully this works? @cbeer